digitalents.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="./client/assets/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles/digitalents.css" />
+    <link rel="stylesheet" href="styles/index.css" />
```

Comment:
links should start with " ./ " always, to tell github where to look for. saying that please change the link address to " ./styles/index.css". The others too with this example.